test/Microsoft.ApplicationInsights.AspNet.Tests/FakeTelemetryChannel.cs

```diff
@@ -6,6 +6,8 @@
     // TODO: Remove FakeTelemetryChannel when we can use a dynamic test isolation framework, like NSubstitute or Moq
```

Comment:
Maybe Moq is working with VS RC. If you will not get to it, I'll try it when will have time